recipe/meta.yaml

```
@@ -1,21 +1,21 @@
 {% set name = "cramjam" %}
-{% set version = "2.6.2" %}
+{% set version = "2.7.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 1ffdc8d1381b5fee57b33b537e38fa7fd29e8d8f3b544dbab1d71dbfaaec3bef
+  sha256: 579fb724eec048b1a18ca8f7ad9a7ef296dc02eba5f87fd4d5031f0c32c5c9ac
 
 build:
   number: 0
-  #skip s390x as it lack maturin dependency
-  skip: True # [py<37 or s390x] 
+  skip: True # [py<37]
   script:
     - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
-
+  missing_dso_whitelist:
+    - $RPATH/ld64.so.1
```

```suggestion
    - $RPATH/ld64.so.1. # [s390x]
```
We want to do this only on s390x.